packages/vm/src/runBlock.ts

```diff
@@ -192,6 +192,13 @@ export async function runBlock(this: VM, opts: RunBlockOpts): Promise<RunBlockRe
     throw err
   }
 
+  let requestsRoot
+  let requests: CLRequest[] | undefined
+  if (block.common.isActivatedEIP(7685)) {
+    const requests = await accumulateRequests(this)
```

Comment:
I think here the `const requests` again flag the `let requests` above (I actually just checked and it does). So the `const` should be removed. (`let requests` stays undefined always)